### ISSUE TITLE
When you create a new terrain, it doesn't show up in the editor

### ISSUE THREAD

**Issue by Ayviesso** [FlaxEngine/FlaxEngine#3115]
**Issue description:**

When you create a new terrain, it doesn't show up in the editor. You can edit it (sculpting), but it will not show up. In 1.8.2 (or something like this. Previous version) dont have this issue. 
<img width="913" alt="Аннотация 2024-12-21 152714" src="https://github.com/user-attachments/assets/bc986c21-bcbd-4d40-8915-705fa796033e" />
<img width="914" alt="Аннотация 2024-12-21 152749" src="https://github.com/user-attachments/assets/d11facef-2675-4f46-ba31-c54a4f396993" />
Maybe I have the same issue as this one: https://github.com/FlaxEngine/FlaxEngine/issues/3050
But I'm not sure and it shows up as fixed.

**Steps to reproduce:**

Install flax editor 1.9 on windows 10 x64, create new basic scene project, create a new terrain without any custom configuration.

**Minimal reproduction project:**

[MyProject.zip](https://github.com/user-attachments/files/18219012/MyProject.zip)

**Flax version:**

1.9

**Comment by xxSeys1** [comment]
This is already fixed on the master branch, see this: https://github.com/FlaxEngine/FlaxEngine/commit/2532a57b0b44720d61f79eae54957abc415c501d

**Comment by Ayviesso** [comment]
I have installed engine today, and all what you see happened today. Do you know how to fix it? Or it just not included into

**Comment by Ayviesso** [comment]
Missclick closed xD

**Comment by Ayviesso** [comment]
I have installed last (daily) version. And deleted all other versions just to be sure. New project, latest version, same issue. I am not sending screenshots because they will be the same. Here is my project: (select terrain in scene menu and find it on a map using Sculpt tool). 
[MyProject.zip](https://github.com/user-attachments/files/18219551/MyProject.zip)

**Comment by Ayviesso** [comment]
Any ideas? xD

**Comment by LordCocoNut** [comment]
This is only partially fixed. The terrain has been fixed in a way its able to show terrain.  I still have to assign some material on my own for example as the terrain is always created without it.

**Comment by Ayviesso** [comment]
Even when I add white material - terrain is invisible.

**Comment by Myterian** [comment]
It needs to be a material in the terrain domain. You can set the domain in the material editor

**Comment by Ayviesso** [comment]
How?...

**Comment by Myterian** [comment]
![grafik](https://github.com/user-attachments/assets/3f63176a-79be-4b20-bd16-1e330eed1476)

**Comment by Myterian** [comment]
No, leave it open. Having a workaround is good, but it's something that should be working by default in engine

**Comment by Tryibion** [comment]
@mafiesto4 , the default material "Engine/DefaultTerrainMaterial" need's its domain changed to be `Terrain`, it is still set as `Surface`.
